src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -12,9 +12,12 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Roslyn.Test.Utilities
 Imports Roslyn.Test.Utilities.TestBase
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module CompilationUtils
 
+    Private ReadOnly Property _pooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool()
```

Comment:
Never mind, the type we are pooling is actually PooledStringBuilder and the name reflects that.

---
In reply to: [102538626](https://github.com/dotnet/roslyn/pull/17134#discussion_r102538626) [](ancestors = 102538626)